From 2204024ef5e610480a62274edbc7beda91e4f053 Mon Sep 17 00:00:00 2001
From: Samuel Debionne <samuel.debionne@esrf.fr>
Date: Wed, 12 Feb 2025 09:06:56 +0100
Subject: [PATCH] Remove boost_python from global dependency list

---
 build.jam | 1 -
 1 file changed, 1 deletion(-)

diff --git a/build.jam b/build.jam
index 39d8d93d..9b90d72f 100644
--- a/build.jam
+++ b/build.jam
@@ -17,7 +17,6 @@ constant boost_dependencies :
     /boost/lexical_cast//boost_lexical_cast
     /boost/mpl//boost_mpl
     /boost/optional//boost_optional
-    /boost/python//boost_python
     /boost/serialization//boost_serialization
     /boost/smart_ptr//boost_smart_ptr
     /boost/static_assert//boost_static_assert
